src/Assets/Pack/Microsoft.AspNetCore.Assets.Pack.csproj

```diff
@@ -0,0 +1,212 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <Description>ASP.NET Core static framework assets</Description>
+    <PackageId>Microsoft.AspNetCore.Assets.Pack</PackageId>
+    <IsPackable>true</IsPackable>
+    <OutputType>Library</OutputType>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <ImplicitUsings>disable</ImplicitUsings>
+
+    <!-- The package doesn't produce any lib or ref assemblies -->
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+
+    <StaticWebAssetBasePath>_framework</StaticWebAssetBasePath>
```

Comment:
We should not set the base path to `_framework` it should be `/` and the relative paths on the assets to include `_framework`.

Reply:
Actually, this should be `$(StaticWebAsetBasePath)`, if the user sets the base path in their project, the assets should be located in their app base path, not anywhere else.

Reply:
Would this require that we move most of the static web asset generation into the `build/*.targets` file so we get insight into the props of the consuming project? Or is there another way you had in mind that we'd achieve this?

Reply:
We don't really need to move the logic there, we can "escape" the value so that it ends up as `$(StaticWebAsetBasePath)` in the file.

Same with the `$(SourceId)`, we can use escaping for now